privileged_residues/chemical.py

```diff
@@ -196,6 +196,9 @@ def _sc_acceptor(pose, selected):
                 is_eneg = reg.search(rsd.atom_name(j))
                 batm = rsd.atom_base(j)
 
+                if (rsd.name() == "OH_" and re.search("OH", rsd.atom_name(j))):
```

Comment:
This should not be in production code.

I will align serines to the hydroxyl groups.